lib/ace/edit_session/bracket_match.js

```diff
@@ -37,6 +37,83 @@ var Range = require("../range").Range;
 
 function BracketMatch() {
 
+
+    this.getBracketsCoor = function(position){
+        if (position.column == 0) return null;
+
+
+        //This part can be combine with while-cycle
+        var charBeforeCursor = this.getLine(position.row).charAt(position.column - 1);
+        var matchLeft = charBeforeCursor.match(/([\(\[\{])/);
+        if(matchLeft){
```

Comment:
When you edit an existing file it makes sense to follow the formatting style used in that file. In ACE code they put a space between if and opening brace and after closing brace and before opening curly bracket -- please do the same.